### PR TITLE
Release version 4.5.1

### PR DESCRIPTION
**Pull request by jack7ellis**
### Description of Changes
Release version 4.5.1

Contains dependency updates and logging changes (reduces the logging severity).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -195,3 +195,8 @@ can get data for both organization and organization brand pages.
 * Update edgeType parameter value for the /networkSizes endpoint to be 
   COMPANY_FOLLOWED_BY_MEMBER instead of CompanyFollowedByMember. 
   This changed in the LinkedIn API 202305 release.
+
+## 4.5.1 (Work in progress)
```
Comment (djjashton) [inline CHANGELOG.md:199]: Merging into master - this needs today's date

**Review by djjashton — CHANGES_REQUESTED** [review]
Need to include a date update as well

File: CHANGELOG.md (hunk 1 above)
Comment (jack7ellis) [reply CHANGELOG.md:199]: Thanks David, made these changes here: https://github.com/ebx/ebx-linkedin-sdk/pull/384

Comment (jack7ellis) [reply CHANGELOG.md]: Thanks for reviewing that, ready to go now

**Review by djjashton — APPROVED** [review]
Brill

**Pull request merged**